local_dev_files/build_dev_env.sh

```diff
@@ -0,0 +1,124 @@
+fi
+
+if aws dynamodb list-tables | grep -q "tfstate-lock"; then
+  printf "${yellowColor}=> Terraform Lock DynamoDB table exists...${reset}\n"
+else
+  printf "${yellowColor}=> Detected fresh AWS account instance.  Setting up Terraform Lock table...${reset}\n"
+
+  fresh_instance=true
+
+  # create DynamoDB table for Terraform state locking
+  aws dynamodb create-table \
+    --table-name tfstate-lock \
+    --attribute-definitions AttributeName=LockID,AttributeType=S \
+    --key-schema AttributeName=LockID,KeyType=HASH \
+    --provisioned-throughput ReadCapacityUnits=5,WriteCapacityUnits=5 \
+    >/dev/null
+fi
+
+if [ "$fresh_instance" = true ]; then
+  printf "${yellowColor}=> Detected fresh AWS account instance.  Clearing local files...${reset}\n"
+  for dir in $basedir/env/cloud/*/; do
+    rm -rf "${dir}.terragrunt-cache"
+    rm -f "${dir}.terraform.lock.hcl"
+  done
+fi
+
+printf "${greenColor}=> Creating AWS services${reset}\n"
+
+printf "${greenColor}...Setting up KMS${reset}\n"
+cd $basedir/env/cloud/kms
+terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
+
+printf "${greenColor}...Setting up Network${reset}\n"
+cd $basedir/env/cloud/network
+terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
+
+printf "${greenColor}...Setting up Secrets Manager${reset}\n"
+cd $basedir/env/cloud/secrets
+terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
+
+printf "${greenColor}...Creating SQS queue${reset}\n"
+cd $basedir/env/cloud/sqs
+terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
+
+printf "${greenColor}...Creating SNS queue${reset}\n"
+cd $basedir/env/cloud/sns
+terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
+
+printf "${greenColor}...Setting up Redis${reset}\n"
+cd $basedir/env/cloud/redis
+terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
+
+printf "${greenColor}...Setting up RDS${reset}\n"
+cd $basedir/env/cloud/rds
+terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
+
+printf "${greenColor}...Setting up S3${reset}\n"
+cd $basedir/env/cloud/s3
+terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
+
+printf "${greenColor}...Setting up DynamoDB${reset}\n"
+cd $basedir/env/cloud/dynamodb
+terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
+
+printf "${greenColor}...Setting up ECR${reset}\n"
+cd $basedir/env/cloud/ecr
+terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
+
+printf "${greenColor}...Setting up Lambdas${reset}\n"
+cd $basedir/env/cloud/lambdas
+terragrunt apply --terragrunt-non-interactive -auto-approve --terragrunt-log-level warn
+
+printf "${greenColor}...Setting up VPN Client${reset}\n"
+
+cd $basedir/aws/vpn/lambda/code
+yarn build && yarn postbuild
```

Comment:
```suggestion
yarn install && yarn build && yarn postbuild
```

Reply:
A `yarn build` should automatically trigger a `yarn install` but we can modify it to be safe.

Reply:
Interesting! Before adding the `yarn install` it was just erroring out saying that `tsc` command was not found.